### ISSUE TITLE
[v1.1.0] no announcements made for several mobs

### ISSUE THREAD

**Issue by gyroplast** [gyroplast/mod-dont-starve-chat-announcements#2]
Affected Versions: Chat Announcements <= 1.1.0

Several mob deaths are not announced at all, due to a mismatch of the config setting name lookup and the actual prefab name. For example, the configuration item for both koalefants is named ```death_koalefant```, but the prefabs are named ```koalefant_summer``` and ```koalefant_winter```, respectively, and when looking up the config item with the actual prefab name, no config info is found, and subsequently no announcement is set up for the "invalid" prefab.

Long story short: really stupid oversight on my part, fortunately with an easy fix.

The affected mobs without announcements are: 
- Celestial Champion
- all three treeguard variants
- Summer and Winter Koalefant
- all shadow chesspieces
- both stalker variants
- Misery Toadstool

Many thanks to [CampbellSoupBoy](https://steamcommunity.com/profiles/76561198007478068) for reporting the issue!
